src/structured-data/structured-data-tags.js

```diff
@@ -56,11 +56,25 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
   });
 }
 
+function storyAccessLevel(accessLevelValue) {
+  if (accessLevelValue === null || accessLevelValue === 100) {
```

Comment:
Don't look accellLevel, look at "access". it can be null, public or subscription